clang/lib/Sema/SemaDeclAttr.cpp

```diff
@@ -1980,6 +1981,23 @@ static void handleWeakRefAttr(Sema &S, Decl *D, const ParsedAttr &AL) {
   D->addAttr(::new (S.Context) WeakRefAttr(S.Context, AL));
 }
 
+// Mark alias/ifunc target as used. For C++, we look up the demangled name
```

Comment:
This should also happen in C because of `__attribute__((overloadable))` (so the code is correct, but we should fix up the comment and add a test case in C for this situation)

Reply:
Thanks for mentioning `overloadable`. Added a test and fixed the comment.